packages/@uppy/svelte/src/components/Dashboard.svelte

```diff
@@ -3,10 +3,13 @@
   import type { Uppy } from '@uppy/core';
   import DashboardPlugin from '@uppy/dashboard'
 
+  type M = any
+  type B = any
```

Comment:
There should be a way for users to specify some specific types. I don't know if that's possible in Svelte components.